lms/services/lti_role_service.py

```diff
@@ -23,6 +23,13 @@ def get_roles(self, role_description: str) -> List[LTIRole]:
         # Pylint is confused about the `in_` for some reason
         roles = self._db.query(LTIRole).filter(LTIRole.value.in_(role_strings)).all()
 
+        for role in roles:
```

Comment:
In some instances we'd might want to do a DB migration and update all roles in one go but I don't see any downside to doing it as we see the roles from the launches.